pkg/operation/botanist/secrets.go

```diff
@@ -796,6 +802,38 @@ func (b *Botanist) DeploySecrets(ctx context.Context) error {
 		b.CheckSums[name] = common.ComputeSecretCheckSum(secret.Data)
 	}
 
+	wildcardCerts := &corev1.SecretList{}
+	if err := b.K8sSeedClient.Client().List(
+		ctx,
+		wildcardCerts,
+		client.InNamespace(v1beta1constants.GardenNamespace),
+		client.MatchingLabels(map[string]string{v1alpha1constants.GardenRole: common.ControlPlaneWildcardCert}),
+	); err != nil {
+		return err
+	}
+
+	if meta.LenList(wildcardCerts) > 1 {
```

Comment:
Any particular reason for using `meta.LenList` and not only `len(wildcardCerts.Items)`?